pallets/capacity/src/lib.rs

```diff
@@ -158,6 +164,21 @@ pub mod pallet {
 	pub type CapacityLedger<T: Config> =
 		StorageMap<_, Twox64Concat, MessageSourceId, CapacityDetails<BalanceOf<T>, T::BlockNumber>>;
 
+	/// Storage for the current epoch number
+	#[pallet::storage]
+	#[pallet::getter(fn get_current_epoch)]
+	pub type CurrentEpoch<T: Config> = StorageValue<_, T::BlockNumber, ValueQuery>;
+
+	/// Storage for the current epoch info
+	#[pallet::storage]
+	#[pallet::getter(fn get_current_epoch_info)]
+	pub type CurrentEpochInfo<T: Config> = StorageValue<_, EpochInfo<T::BlockNumber>, ValueQuery>;
+
+	/// Storage for the current epoch number
+	#[pallet::storage]
+	#[pallet::getter(fn get_current_epoch_used_capacity)]
+	pub type CurrentEpochUsedCapacity<T: Config> = StorageValue<_, BalanceOf<T>, ValueQuery>;
```

Comment:
maybe for CurrentEpochInfo and CurrentEpochUsedCapacity too.

Reply:
ah you mean whitelist this storage too.